server/sonar-web/src/main/webapp/WEB-INF/app/controllers/sessions_controller.rb

```diff
@@ -41,10 +41,7 @@ def login
     begin
       self.current_user = User.authenticate(params[:login], params[:password], servlet_request)
       if logged_in?
-        if params[:remember_me] == '1'
-          self.current_user.remember_me
-          cookies[:auth_token] = { :value => self.current_user.remember_token , :expires => self.current_user.remember_token_expires_at, :http_only => true }
-        end
+        set_user_session
```

Comment:
In fact it's not related to the removal of remember me cookie : we need to set the user session here otherwise we won't be able to know if the user is connected in GenerateJwtTokenFilter